src/app/services/auth.service.ts

```diff
@@ -38,6 +39,16 @@ export class AuthService {
     );
   }
 
+  register(userRegisterDto: IUserRegisterDto): Observable<ITokenDto> {
+    userRegisterDto.clientUri = `${environment.web.protocol}://${environment.web.baseUrl}/registerConfirmation`;
+    return this.http.post<ITokenDto>(`${this.baseUrl}/register`, userRegisterDto)
+    .pipe(
+      catchError(error => {
+        return throwError(error);
```

Comment:
Dieser CatchError tut nix, sondern wirft nur den Fehler wieder :) Den kannst du rausnehmen

Reply:
Das gleiche übrigens in der Login Methode